### ISSUE TITLE
installation instructions improvment 

### ISSUE THREAD

**Issue by Neetuj** [softlayer/knife-softlayer#20]
When i installed knife-soflayer i ran into those issues ( so i paln to create a PR for this soon) 

1) The default location in which knife expects to find this file is ~/.chef/knife.rb

2) and if you run into installation error like these :- 
 "Building nokogiri using packaged libraries. checking for gzdopen() in -lz... no zlib is missing; necessary for building libxml2 *** extconf.rb failed ***"

  then try :- 
  sudo apt-get install zlib1g-dev
   gem install nokogiri
  gem install knife-softlayer

**Comment by cphrmky** [comment]
I'd love to see a PR with updated documentation.

Your fix though is specific to the platform and version you're working from, and nokogiri causes problems on basically every platform somewhat regularly.  

When you put up the PR let's do an alternative recommendation that should fix that problem across most common platforms?  Maybe something like this?

```bash
export NOKOGIRI_USE_SYSTEM_LIBRARIES=1
gem install knife-softlayer
```

Thx @Neetuj

**Comment by Neetuj** [comment]
yes .. http://www.nokogiri.org/tutorials/installing_nokogiri.html
